scripts/globals/spells/black/drain.lua

```diff
@@ -41,7 +41,8 @@ spellObject.onSpellCast = function(caster, target, spell)
         dmg = 0
     end
 
-    if target:isUndead() or target:hasImmunity(xi.immunity.DRAIN) then
+    -- ID 2156: Jormungand
+    if target:isUndead() or target:getPool() == 2156 then
```

Comment:
Jorm isn't actually immune to drain. I was just adding in the immunity because I thought it existed. Can just completely ignore drain for this.